core/src/main/java/org/apache/carbondata/core/datamap/DataMapChooser.java

```diff
@@ -121,6 +121,10 @@ public DataMapExprWrapper chooseCGDataMap(FilterResolverIntf resolverIntf) {
     return chooseDataMap(DataMapLevel.CG, resolverIntf);
   }
 
+  public boolean hasCGDataMap() {
```

Comment:
It is strange we add this func in a class called `Chooser`, can you put in a more suitable class?